pokemongo_bot/cell_workers/pokemon_catch_worker.py

```diff
@@ -167,8 +182,13 @@ def work(self):
                                         logger.log(
                                             '[#] {} has been exchanged for candy!'.format(pokemon_name), 'green')
                                     else:
-                                        logger.log(
-                                        '[x] Captured {}! [CP {}]'.format(pokemon_name, cp), 'green')
+                                        logger.log('[x] Captured {}! [CP {}] [{}/{}/{}]'.format(
```

Comment:
If you are going to remove one, this one should probably be kept and moved up. Otherwise this is only printed if we don't release or evolve